sources/github/graphql.js

```diff
@@ -191,3 +203,59 @@ module.exports.updatedIssuesByRange = updatedByRange({
     }
   `,
 })
+
+module.exports.reposByRange = searchByRange({
+  type: 'REPOSITORY',
+  qualifier: 'pushed',
```

Comment:
I completely forgot why we parameterized this in the REST API version and now I recall :trollface:

Reply:
ya was wondering why you didn't lol